tf_keras/applications/applications_test.py

```diff
@@ -138,7 +150,16 @@
 
 
 class ApplicationsTest(tf.test.TestCase, parameterized.TestCase):
-    def assertShapeEqual(self, shape1, shape2):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
This should be:

```
    def setUp(self):
        self.original_image_data_format = backend.image_data_format()
```

The reason is that you want this to be called between each test method. Otherwise some test methods that do not set the image data format will be run with some random value, which will fail for models that do not support `channels_first`. Right now, the tests are failing because of this.

Reply:
Thank you, I have made the fix